src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    divide(num) {
+      this.result /= num;
+    },
+
+    multiply(num) {
+      this.result *= num;
+    },
```

Comment:
The `multiply` method is properly using the `*=` operator as suggested by the [CODE STYLE] checklist.